src/app.tsx

```diff
@@ -37,18 +43,26 @@ class App extends React.Component<{}, TilgangTilBrukerState> {
 
 
     public componentWillUnmount(){
-        this.stopEventListening();
+        this.stopAktivitetsplanEventListening();
+        this.stopMaoEventListening();
     }
 
+    startMaoEventListening() {
+        getWindow().addEventListener('rerenderMao', () => this.setState({maoKey: this.state.maoKey + 1}))
```

Comment:
Kan bruke setState med funkjson for å sikre at verdien blir satt riktig

Reply:
hvordan fungerer det hvis addEventListner trenger callback?